schedule/hpc/single_machine_test.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: single machine HPC tests
+description:    >
+     Maintainer: schlad
+     all singe machine HPC tests based loaded based on hpctest
+vars:
+  DESKTOP: textmode
+  INSTALLONLY: 1
```

Comment:
INSTALLONLY=1 is needed for installation jobs only

Reply:
ah sure. re-sent.